core/src/main/java/org/lflang/target/property/DNETProperty.java

```diff
@@ -0,0 +1,25 @@
+package org.lflang.target.property;
+
+/**
+ * If true, the RTI sends DNET signals during the execution of federation. The default is ture (for
+ * now, for testing).
```

Comment:
```suggestion
 * @brief Target property turning on or off the DNET signal optimization.
 * 
 * If this target property is true, the RTI sends DNET (downstream next event tag) signals to an upstream federate
 * to tell the federate that sending LTC and NET signals with tags less than the specified value is unnecessary.
 * The default is true.
```
Note that if we make the default value `false`, this feature will not be adequately tested. Is there any reason it should not be `true` by default?

Reply:
I updated the comment in [83bd58f](https://github.com/lf-lang/lingua-franca/pull/2400/commits/83bd58f7eeb5a8a0e5234ab299186f6a9ebf16b0). 

I'm concerned that if every reaction sends a message every time it's triggered, the DNET may be useless. I'm trying to run some applications to see whether the impact is negligible, but I'm struggling to generalize the cases.

I'll make sure to show you a convincing result shortly.